README.md

```diff
@@ -5,6 +5,8 @@
 [![Doxygen -> gh-pages](https://github.com/BlueBrain/HighFive/workflows/gh-pages/badge.svg)](https://BlueBrain.github.io/HighFive)
 [![codecov](https://codecov.io/gh/BlueBrain/HighFive/branch/master/graph/badge.svg?token=UBKxHEn7RS)](https://codecov.io/gh/BlueBrain/HighFive)
 
+Documentation: https://bluebrain.github.io/HighFive/
```

Comment:
I think all this could be phrased nicer, but maybe we should give the README a holistic make-over some time later. Having the link in plain sight is a better than having to decipher which badge to click, though.

Reply:
Yes, I saw the badge but it seems me complicate to see and understand you should click on it to see the documentation